packages/core/src/components/ButtonGroup/types.ts

```diff
@@ -0,0 +1,12 @@
+import { SerializedStyles } from '@emotion/react';
+
+export type Item = {
```

Comment:
Let's rename this to ButtonGroupItem. 
If we decide to export this type, we'll likely have a duplicate name (`Item`).

Reply:
Done